### PR TITLE
Raise an error when a user tries to update a signing service

### PR DESCRIPTION
**Pull request by lubosmj**
closes #7701

### CODE REVIEW

**Comment by lubosmj** [comment]
When a user performs a single update, the error is correctly raised. However, this approach has a limitation when executing bulk operations (such as SigningService.objects.filter(name="s").update(name="b")):

> Be aware that the update() method is converted directly to an SQL statement. It is a bulk operation for direct updates. It doesn’t run any save() methods on your models, or emit the pre_save or post_save signals (which are a consequence of calling save()), or honor the auto_now field option. If you want to save every item in a QuerySet and make sure that the save() method is called on each instance [https://docs.djangoproject.com/en/2.2/topics/db/queries/#updating-multiple-objects-at-once].

File: pulpcore/app/models/content.py [hunk 1]
```diff
@@ -609,6 +610,13 @@ def save(self, *args, **kwargs):
         self.validate()
         super().save(*args, **kwargs)
 
+    @hook(BEFORE_UPDATE)
+    def on_update(self):
+        raise RuntimeError(
+            "The signing service is immutable. It is advised to create a new signing service "
```
Comment (bmbouter) [inline pulpcore/app/models/content.py:616]: Can this be i8ln with gettext please?

**Review by bmbouter — CHANGES_REQUESTED** [review]
This looks good once the i8ln is fixed. Thank you!

**Review by mdellweg — APPROVED** [review]
The requested change has been made.

**Review by bmbouter — APPROVED** [review]
Thank you!

**Pull request merged**